hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/XceiverServerRatis.java

```diff
@@ -200,6 +200,9 @@ private RaftProperties newRaftProperties() {
     TimeUnit timeUnit;
     long duration;
 
+    // set the node failure timeout
+    setNodeFailureTimeout(properties);
```

Comment:
`setNodeFailureTimeout` initializes a field. I think we can initialize the field in the constructor itself and remove this function.